core/SettingsPiwik.php

```diff
@@ -427,6 +427,10 @@ public static function overwritePiwikUrl($currentUrl)
      */
     public static function isHttpsForced()
     {
+        if (!SettingsPiwik::isPiwikInstalled()) {
```

Comment:
it's to make sure that user is not redirected to HTTPS when it may not be working yet, ie. while installing Piwik, in case user uses  `common.config.ini.php` config file which may have `force_ssl` = 1